tests/framework/db/ActiveRecordTest.php

```diff
@@ -1893,24 +1893,24 @@ public function testLegalValuesForFindByCondition($modelClassName, $validFilter)
     public function illegalValuesForFindByCondition()
     {
         return [
-            [Customer::className(), ['id' => ['`id`=`id` and 1' => 1]]],
```

Comment:
Associative arrays are not allowed anymore in PHP 8 for `ReflectionMethod::invokeArgs()` hence the change.

Reply:
Umm... maybe we can convert that somewhere in the middle instead of changing current behavior?

Reply:
But how is it changed? As far as I can see the output was using only array value anyway and that artificial keys (like `id`) were unneeded. Unless I don't know something here...

Reply:
Need to review it again. Anyway, big thanks for looking into it.